server/controllers/entities/lib/properties/properties_config_bases.ts

```diff
@@ -77,3 +79,14 @@ export const imageHash = {
   validate: ({ value }: { value: string }) => isImageHash(value),
   uniqueValue: true,
 } as const
+
+export const entityType = {
+  datatype: 'entity-type',
+  primitiveType: 'string',
+  format: trim,
+  validate: ({ value }) => {
+    return allLocallyEditedEntitiesTypes.includes(value)
+  },
+  adminEditOnly: true,
```

Comment:
It would be more adaptable to directly have the access level written here:
```suggestion
  editAccessLevel: 'admin',
```
This would need to transform PR occurrences of `userIsAdmin: userHasAdminRole(user)` -> `userAccessLevel: getUserAccessLevels(user)`

Reply:
It might be YAGNI, but I gave it a try in 66db7d818